tests/foreman/ui/test_contenthost.py

```diff
@@ -172,7 +172,7 @@ def test_positive_end_to_end(session, repos_collection, vm):
     """Create all entities required for content host, set up host, register it
     as a content host, read content host details, install package and errata.
 
-    :id: f43f2826-47c1-4069-9c9d-2410fd1b622c
```

Comment:
Did fix_uuids make this change? Is it because of copying this file for some testing/debug/dev work? I know you had brought that up in chat, and I don't think changing this UUID should be required unless we've created a duplicate somehow in master branch?

Reply:
Removed that accidental change.